javascript/javascript2/week3/game/main.js

```diff
@@ -0,0 +1,119 @@
+const btn = document.getElementById("button");
+//this element will be used to show messages to user
+const pUserMessage = document.getElementById("message");
+let timer = 0;
+
+//adding event listener to button to start/reset the game
+btn.addEventListener("click", () => {
+    //getting the value of sec from user
+    timer = document.getElementById("time").value;
+    if (!timer) {
+        pUserMessage.innerHTML = "Set timer to start the game";
+    } else {
+        //button will work both to start game and reset game
+        if (btn.value === "Reset game") {
+            resetGame();
+        } else {
+            startGame(timer);
+        }
+    }
+});
+
+//variables to hold key press count
+let countS = 0;
+let countL = 0;
+const pCountS = document.getElementById("countS");
+const pCountL = document.getElementById("countL");
+//this varible holds funciton to count
+const countKeyPress = (event) => {
+    if (event.key === "s") {
+        countS++;
+        pCountS.innerHTML = countS;
+    }
+    if (event.key === "l") {
+        countL++;
+        pCountL.innerHTML = countL;
+    }
+}
+
+//this variable will be used to hold countdown interval to show to user
+let countDown;
+//this variable will be used to hold Timeout to end Game
+let endGame;
+
+//on start game set the Event listner and count,end game after timeout
+function startGame(timer) {
```

Comment:
This method is doing quite a lot, might be worth taking a look into if this can be split into smaller methods